src/Sylius/Behat/Context/Api/Admin/ManagingShippingMethodsContext.php

```diff
@@ -403,14 +403,15 @@ public function iShouldSeeShippingMethodsInTheList(int $count): void
     }
 
     /**
-     * @Then the shipping method :name should be in the registry
-     * @Then the shipping method :name should appear in the registry
+     * @Then the shipping method :shippingMethod should be in the registry
+     * @Then the shipping method :shippingMethod should appear in the registry
+     * @Then the :shippingMethod shipping method should be successfully created
      */
-    public function theShippingMethodShouldAppearInTheRegistry(string $name): void
+    public function theShippingMethodShouldAppearInTheRegistry(ShippingMethodInterface $shippingMethod): void
```

Comment:
if nothing more than the name is needed in this method, then this change is not necessary, because this way we additionally query the database